.azure-pipelines/client.yml

```diff
@@ -108,7 +108,7 @@ jobs:
 
       - task: DotNetCoreCLI@2
         displayName: 'Build & Test (no live tests)'
-        condition: eq(variables['System.TeamProject'], 'public')
+        condition: eq(variables['System.TeamProject'], 'playground')
```

Comment:
I assume this is temporary?

Reply:
Yes, I am just testing it on the playground